docs/lineage/airflow.md

```diff
@@ -99,6 +100,7 @@ If you are looking to run Airflow and DataHub using docker locally, follow the g
    [lineage]
    backend = datahub_provider.lineage.datahub.DatahubLineageBackend
    datahub_kwargs = {
+       "enabled": true,
```

Comment:
Will this work with the plugin as well? -> https://github.com/datahub-project/datahub/tree/master/metadata-ingestion-modules/airflow-plugin

Reply:
updated so that it does